src/evaluator/evaluator/eufs_adapter.py

```diff
@@ -78,6 +80,17 @@ def __init__(self, node: rclpy.node.Node):
 
         self._perception_time_sync_.registerCallback(self.perception_callback)
 
+        self._planning_time_sync_ = message_filters.ApproximateTimeSynchronizer(
+            [
+                self.node.planning_subscription_,
+                self.node.planning_gt_subscription_,
+            ],
+            10,
+            0.5,
+        )
+
+        self._planning_time_sync_.registerCallback(self.planning_callback)
+
```

Comment:
I think this is not necessary as the planning output and the mocker node subscriptions don't need to be in the adapter